service/registrationservice.php

```diff
@@ -215,7 +215,7 @@ public function checkAllowedDomains($email) {
 			foreach ($allowed_domains as $domain) {
 				$maildomain = explode("@", $email)[1];
 				// valid domain, everythings fine
-				if ($maildomain === $domain) {
+				if (strtolower($maildomain) === strtolower($domain)) {
```

Comment:
Better would be to use `strcasecmp` to avoid the temporary string objects:
https://www.php.net/manual/en/function.strcasecmp.php
```
if (strcasecmp($maildomain, $domain) === 0) {
```